packages/docusaurus/src/commands/start.ts

```diff
@@ -103,7 +103,7 @@ export default async function start(
     cwd: siteDir,
     ignoreInitial: true,
     usePolling: !!cliOptions.poll,
-    interval: Number.isInteger(cliOptions.poll)
+    interval: Number.isInteger((cliOptions.poll as unknown) as never)
```

Comment:
what was the problem?

Reply:
I thought I was going to need to update this for this PR, and my IDE was complaining about it.  I'll revert if you want.

Reply:
As it's not strictly related to the PR and I don't understand the intent, I'd prefer removing this line and discuss it as a separate issue/PR if needed

Reply:
Sure.

Reply:
Done.